python-main.js

```diff
@@ -648,6 +665,9 @@ function web_editor(config) {
                         fileReader.readAsArrayBuffer(file);
                     });
                     inputFile.value = '';
+
+                    var content = $('.expandable-content')[0];
+                    content.style.maxHeight = (content.scrollHeight + 200) + "px";
```

Comment:
Where is 200 coming from? If there is a DOM element height we can query instead of adding a magic number here then that would be better.

Reply:
scrollHeight is supposed to give the the height of the element including padding and its overflow. However, just using that on its own appears to cut off content when you start adding files. This magic number was my workaround for that, I am still trying to figure out why scrollHeight isn't giving me the true height of the element.

Reply:
Have you been able to look into this one again?